app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="row">
+    <div class="mt-4 col-3"> 
+        <% if @user.image? %>
+             <%= image_tag "http://graph.facebook.com/#{@user.uid}/picture?type=large", class: "avatar-scale" %>
+        <% else %>
+             <%= image_tag gravatar_for(@user), class: "avatar-scale"%>
+        <% end %>
+
+        <div class="card">
+            <div class="card-header text-white">Verification</div>
+            <div class="card-body">
+              <ul class="sidebar-list">
+                    <li>Facebook<span class="pull-right icon-babu"><i class="fa fa-check-circle"></i></span></li>
```

Comment:
Display Email if user signed-up through email, and facebook if user signed-up through facebook.